habitat-baselines/habitat_baselines/agents/ppo_agents.py

```diff
@@ -23,22 +24,25 @@
 from habitat_baselines.utils.common import batch_obs
 
 
-def get_default_config() -> Config:
-    c = Config()
-    c.INPUT_TYPE = "rgb"
-    c.MODEL_PATH = "data/checkpoints/gibson-rgb-best.pth"
-    c.RESOLUTION = 256
-    c.hidden_size = 512
-    c.RANDOM_SEED = 7
-    c.PTH_GPU_ID = 0
-    c.goal_sensor_uuid = "pointgoal_with_gps_compass"
-    return c
+@dataclass
+class PPOAgentConfig:
+    INPUT_TYPE: str = "rgb"
```

Comment:
I think it would actually be better not to. Some of these keys are the same as the ones of habitat-lab config, keeping them upper case makes it easy to see the keys are different.